vendor/vendor.json

```diff
@@ -69,5 +69,5 @@
 			"revisionTime": "2017-05-12T22:20:15Z"
 		}
 	],
-	"rootPath": "github.com/shomali11/slacker"
+	"rootPath": "github.com/evandroflores/slacker"
```

Comment:
Was this an intentional update? I thought it had to match the origin of the repository.